content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Front end Overhaul of the OpenTelemetry Demo (Go to Next.js)
```

Comment:
"Frontend", "Front-end" or "Front end"?

https://en.wikipedia.org/wiki/Front-end_web_development

Reply:
I think I saw "back-end" used a lot across our docs, so it would be "Front-end" then

Reply:
Sold!